stingray/deadtime/fad.py

```diff
@@ -344,7 +414,7 @@ def get_periodograms_from_FAD_results(FAD_results, kind='ptot'):
         elif '2' in kind:
             powersp.nphots = FAD_results.meta['nph2']
     elif kind == 'cs':
-        powersp = AveragedCrossspectrum(power_type='real')
+        powersp = AveragedCrossspectrum(power_type='all')
```

Comment:
why this change?

Reply:
Because it seemed wrong to throw away half of the information without asking the user